src/Components/Auction/PostRegistrationModal.tsx

```diff
@@ -35,6 +36,26 @@ const RegistrationPending: ModalContent = ({ onClick }) => {
     </>
   )
 }
+
+const RegistrationPendingUnverified: ModalContent = ({ onClick }) => {
+  return (
+    <>
+      <RegistrationPendingHeader />
+      <Serif my={3} size="3t">
+        This auction requires Artsy to verify your identity before bidding.
+        <br />
+        <br />
+        For details about identity verification, please see the FAQ or contact
```

Comment:
Is `FAQ` supposed to be a link here?

Reply:
It should be a link, added a todo to https://github.com/artsy/force/pull/5392.